### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by gewarren**
- make breaking changes template come last in list
- add custom config file for links to other feedback sites

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: docs.microsoft.com site feedback
+    url: https://github.com/MicrosoftDocs/feedback/issues/new/choose
+    about: Please log general docs.microsoft.com site issues here.
```
Comment (mairaw) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    about: Log general docs.microsoft.com site issues here.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+  - name: .NET Core issues
+    url: https://github.com/dotnet/runtime/issues
+    about: Please log .NET Core product issues here.
```
Comment (mairaw) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    about: Log .NET Core product issues here.
```

**Review by mairaw — COMMENTED** [review]
Do you know how this is going to render? One of the problems I see with this is that our repo covers products that are hosted in multiple repos: runtime, machinelearning, winforms, wpf, sdk, and many others. That's why for product feedback we chose https://developercommunity.visualstudio.com/spaces/61/index.html as the main product feedback destination with a few exceptions.

**Comment by gewarren** [comment]
> Do you know how this is going to render? One of the problems I see with this is that our repo covers products that are hosted in multiple repos: runtime, machinelearning, winforms, wpf, sdk, and many others. That's why for product feedback we chose https://developercommunity.visualstudio.com/spaces/61/index.html as the main product feedback destination with a few exceptions.

It will look something like [this](https://github.com/PowerShell/PowerShell/issues/new/choose) (see the last three links).

Do you want me to leave product feedback issues out altogether, or change the URL, or?

**Comment by mairaw** [comment]
Cool, hadn't seen this in action. So probably linking to https://developercommunity.visualstudio.com/spaces/61/index.html would be enough?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: docs.microsoft.com site feedback
+    url: https://github.com/MicrosoftDocs/feedback/issues/new/choose
+    about: Please log general docs.microsoft.com site issues here.
+  - name: .NET Core issues
+    url: https://github.com/dotnet/runtime/issues
```
Comment (gewarren) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
    url: https://developercommunity.visualstudio.com/spaces/61/index.html
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: docs.microsoft.com site feedback
+    url: https://github.com/MicrosoftDocs/feedback/issues/new/choose
+    about: Please log general docs.microsoft.com site issues here.
+  - name: .NET Core issues
```
Comment (gewarren) [inline .github/ISSUE_TEMPLATE/config.yml]:
```suggestion
  - name: .NET platform and other product issues
```

**Comment by gewarren** [comment]
@mairaw Could you approve this? I think the validation will always fail on the template files.

**Comment by mairaw** [comment]
Yes, we need to request @meganbradley to ignore .GITHUB folder.

**Review by mairaw — APPROVED** [review]

**Pull request merged**